PurchasesCoreSwift/Networking/ETagAndResponseWrapper.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ETagAndResponseWrapper.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 6/11/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+internal struct ETagAndResponseWrapper {
+
+    static let eTagKey = "eTag"
+    static let statusCodeKey = "statusCode"
+    static let responseObjectKey = "responseObject"
+
+    let eTag: String
+    let statusCode: Int
+    let responseObject: [String: Any]
+
+    func asDictionary() -> [String: Any] {
+        [
+            ETagAndResponseWrapper.eTagKey: eTag,
+            ETagAndResponseWrapper.statusCodeKey: statusCode,
+            ETagAndResponseWrapper.responseObjectKey: responseObject
+        ]
+    }
```

Comment:
should I do it? We don't use it anywhere else in the code. I am happy to be first

Reply:
so I tried, but [String: Any] does not conform to Encodable. What would you do? I was thinking maybe storing Data instead of [String: Any], but I wonder if you had any other better suggestion.